app/views/kanaui/dashboard/index.html.erb

```diff
@@ -19,13 +19,13 @@
                     <div class="form-group">
                       <div class="col-md-7">
                         <% # TODO datepicker breaks :hover %>
```

Comment:
IIRC I had hacked this because it was breaking the behavior on mouse hover. Can you confirm the mouse hover works / the datepicker experience is the same on other screens? If so, good to merge (just remove the comments).